x-pack/plugins/observability_solution/observability/server/lib/rules/custom_threshold/custom_threshold_executor.test.ts

```diff
@@ -184,63 +271,63 @@ describe('The custom threshold alert type', () => {
     test('alerts as expected with the > comparator', async () => {
       setResults(Comparator.GT, [0.75], true);
       await execute(Comparator.GT, [0.75]);
-      expect(mostRecentAction(instanceID)).toBeAlertAction();
+      expect(getLastReportedAlert(instanceID)).toBeAlertAction();
```

Comment:
I am a bit confused now, is this checking an alert or action? It seems we are getting an alert (getLastReportedAlert) but checking an action (toBeAlertAction).

Reply:
The old implementation causes this confusion.
We used to create an alert and then schedule actions for each actionGroups (alert, warning, recovered etc.)
Now we just report an alert with action group, no separate action scheduling.

Maybe we can rename `toBeAlertAction` to `toHaveAlertAction`?

Reply:
> Maybe we can rename `toBeAlertAction` to `toHaveAlertAction`?

I like this suggestion 👍🏻

Reply:
done.